aws/alb.go

```diff
@@ -331,11 +335,23 @@ func findManagedLoadBalancers(svc elbv2iface.ELBV2API, clusterID string) ([]*Loa
 	return loadBalancers, err
 }
 
+func getListeners(alb elbv2iface.ELBV2API, loadBalancerARN string) ([]*elbv2.Listener, error) {
+	// TODO: paged results
```

Comment:
I couldn't easily find in the docs what the default page size is. I'd expect that we should get, at most, 2 listeners - HTTP and HTTPS. A safer way would not to rely on this and be sure to go through all of them. The only way to guarantee it would be through pagination which the API actually supports - http://docs.aws.amazon.com/sdk-for-go/api/service/elbv2/#ELBV2.DescribeListenersPages